examples/requirements_develop.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+
+# Installs wheels for current develop version 
+import sys, os, subprocess
+parent_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+os.chdir(parent_dir)
+sys.path.insert(1, parent_dir)
+import find_version
+
+# Get package version
+version = find_version.get_package_version()
+git_commit = subprocess.check_output(['git', 'rev-parse', 'HEAD']).decode('UTF-8').strip()
```

Comment:
I think we should do the following to install depthai lib:
`if not gitContext or branch == main then install latest depthai release`, (e.g. -U depthai if we choose to remove version info from development builds)
`else install depthai+commit`

Also rename to `install_requirements.py` and add 3rdparty dependencies to a list, so it will be easier to add new ones. (Similar to gen1)